docs/src/main/sphinx/security/ldap.rst

```diff
@@ -243,15 +244,18 @@ Environment configuration
 TLS configuration
 ~~~~~~~~~~~~~~~~~
 
-Access to the Trino coordinator should be through HTTPS when using LDAP
-authentication. The Trino CLI can use either a :ref:`Java Keystore
-<server_java_keystore>` file or :ref:`Java Truststore <cli_java_truststore>`
-for its TLS configuration.
+When using LDAP authentication, access to the Trino coordinator should be
```

Comment:
is required to ...

Reply:
or "must be"

Reply:
either is fine .. but not "should" "could" "would" ...